src/components/basic/form/CheckedInputGroup.js

```diff
@@ -1,31 +1,29 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React from "react";
+import PropTypes from "prop-types";
 
-const CheckedInputGroup = (props) => {
-	const { heading, hint, describedBy, children } = props;
+const CheckedInputGroup = props => {
+  const { title, hint, describedBy, children } = props;
 
-	return (
-		<fieldset className="dg_fieldset" aria-describedby={describedBy}>
-			<legend className="dg_fieldset-legend">
-				<h1 className="dg_fieldset-heading">{heading}</h1>
-			</legend>
-			{hint && (
-				<span id={describedBy} className="dg_fieldset-hint">
-					{hint}
-				</span>
-			)}
-			{children}
-		</fieldset>
-	);
+  return (
+    <fieldset className="dg_fieldset" aria-describedby={describedBy}>
```

Comment:
@theilman7 I made a quick cahnge for you here, so we could get this completed. Since describedBy insn't requried, i set a default value for describedBy like this.

```js
const ariaDescribedBy = describedBy || title.replace(/\s/g, "");
```

Reply:
this just replaces all the spaces in the title